tools/cvm-image-rewriter/README.md

```diff
@@ -136,19 +136,22 @@ Before running the tool, please choose the plugins that are needed.You can skip
 For example:
 
     ```
-    touch pre-stage/01-resize-image/NOT_RUN
+    touch plugins/01-resize-image/NOT_RUN
     ```
 
 If the guest image is used for CCNP deployment, it's recommended to run below plugin combination according to different initial guest image type.
 Others are not required by CCNP and can be skipped.
-|  Base image | 01  | 02  | 03  | 04  | 05 | 07 | 08 | 09 | 60 | 98 |
+|  Base image | 01  | 02  | 03  | 04  | 05 | 06 | 07 | 08 | 60 | 98 |
 |---|---|---|---|---|---|---|---|---|---|---|
-|  Ubuntu base image | | | | | Y| | Y| Y| | |
+|  Ubuntu base image | | | | | | Y| Y| Y| | |
 | TD enlightened image | | | | | | | Y| Y| | |
 
 **NOTE:**
-  - TD enlightened image means the image already has TDX kernel. If not, plugin 05 is needed to install TDX kernel.
-  - Plugin 08 and 09 prepares device permission for CCNP deployment.
+  - All plugins need to be executed in numerical order.
```

Comment:
Is it the need to be or is it that they will be?

Reply:
Thanks for your review. It is need to be and it will be, for example, plugin 98 cannot be the first one to execute.